Documentation/contributing/release/backports.rst

```diff
@@ -140,6 +140,13 @@ Creating the backports branch
       ...
       $ contrib/backporting/cherry-pick <newest-commit-sha>
 
+   Conflicts may be resolved by applying changes or preferably backporting other
+   PRs to completely avoid them. What is the PR that merged a commit into the
+   master branch can be found via the github UI by inspecting the commit (see `this
+   stackoverflow.com question
+   <https://stackoverflow.com/questions/17818167/find-a-pull-request-on-github-where-a-commit-was-originally-created>`_
+   for more details).
+
```

Comment:
It might also be good to add something like:

```
If the conflict is resolved by modifying a commit during backport, describe the changes made in the commit message and collect these to add to the backport PR description when creating the PR below. This helps to direct backport reviewers towards which changes may deviate from the original commits to ensure that the changes are correctly backported.
```

Reply:
Makes perfect sense.  Added it as well. Thanks!